src/schemas/execution.ts

```diff
@@ -0,0 +1,22 @@
+import mongoose, { Schema } from 'mongoose';
+import { ExecutionDocument, ExecutionModel } from '../types/db/execution.types';
+
+const ExecutionSchema = new Schema({
+    excercise: { type: {
```

Comment:
```
excercise: { type: {
        _id: mongoose.Types.ObjectId,
        name: String,
        instruction: String,
        gifUrl: String,
        muscle: String,
        equipment: String
    }, required: true }
```

bitte ändern zu 

```
excercise: {
        name: String,
        instruction: String,
        gifUrl: String,
        muscle: String,
        equipment: String
    }
```

außerdem kann man in den subdocuments das _id rausschmeißen das brauch man eh nicht ;)